docker-compose-openelis.yml

```diff
@@ -0,0 +1,229 @@
+                      -Djavax.net.ssl.keyStoreType=pkcs12"
+      # Config variables loaded through Tomacat server.xml
+      CATALINA_OPTS: "-Dhapi.ssl.truststorepath=${SSL_TRUSTSTORE_PATH} -Dhapi.ssl.truststorepassword=${SSL_TRUSTSTORE_PASSWORD} -Dhapi.ssl.keystorepath=${SSL_KEYSTORE_PATH} -Dhapi.ssl.keystorepassword=${SSL_KEYSTORE_PASSWORD}"
+      # Config variables loaded through Hapi application.yml
+      FHIR_DATASOURCE_URL : "jdbc:postgresql://db.openelis.org:5432/clinlims?currentSchema=clinlims"
+      FHIR_DATASOURCE_USERNAME: "clinlims"
+      FHIR_DATASOURCE_PASSWORD: ${OE_DB_PASSWORD}
+      FHIR_SERVER_ADRESS: "http://fhir.openelis.org:8080/fhir/"
+
+    logging: *local-logging
+    volumes:
+      -  key_trust-store-volume:/etc/openelis-global
+
+  frontend.openelis.org:
+    image: itechuw/openelis-global-2-frontend:develop
+    container_name: openelisglobal-front-end
+    platform: linux/amd64
+    networks:
+      - ozone
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    logging: *local-logging
+    tty: true
+
+  openelis.proxy:
+    image: itechuw/openelis-global-2-proxy:develop
+    container_name: openelisglobal-proxy
+    platform: linux/amd64
+    ports:
+      - 8090:80
+      - 443:443
+    volumes:
+      - certs-vol:/etc/nginx/certs/
+      - keys-vol:/etc/nginx/keys/
+      - ../../distro/binaries/openelis/volumes/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      # - /KEYSTORE_PASSWORD:/etc/nginx/private/key_pass
+      # - /etc/openelis-global/nginx.cert.pem:/etc/nginx/certs/cert.crt
+      # - /etc/openelis-global/nginx.key.pem:/etc/nginx/certs/cert.key
+    restart: unless-stopped
+    networks:
+      - ozone
+    logging: *local-logging
+    depends_on:
+      - certs
+
+  # astm-http-bridge:
+  #     container_name: astm-http-bridge
+  #     image: itechuw/astm-http-bridge:2.3.6
+  #     platform: linux/amd64
+  #     pull_policy: always
+  #     ports:
+  #         - "8445:8443"
+  #         - "12001:12001"
+  #     volumes:
+  #         - ../../distro/binaries/openelis/volumes/astm/configuration.yml:/app/configuration.yml
+  #         -  key_trust-store-volume:/etc/openelis-global
+  #     environment:
+  #         SPRING_PROFILE: dev
+  #         LOGGING_LEVEL_ORG_ITECH: TRACE
+  #         JAVA_OPTS: |
+  #             -Djavax.net.ssl.trustStore=/etc/openelis-global/truststore
+  #             -Djavax.net.ssl.trustStorePassword=tspass
+  #             -Djavax.net.ssl.trustStoreType=pkcs12
+  #     logging: *local-logging
+  #     networks:
+  #         - ozone
+  #     healthcheck:
+  #         test: ["CMD", "/app/healthcheck.sh"]
+  #         timeout: 10s
+  #         interval: 30s
+  #         retries: 3
+  #         start_period: 2m
+
+
+  autoheal:
+    container_name: autoheal-oe
+    image: willfarrell/autoheal:1.2.0
+    tty: true
+    restart: always
+    networks:
+      - ozone
+    environment:
+      AUTOHEAL_CONTAINER_LABEL: all
```

Comment:
Look like this applies to all services, how does it impact the other services?
If this is helpful, probably we should add it as a common service in `docker-compose-common.yml`